lib/public/AppFramework/Http/Response.php

```diff
@@ -238,6 +238,7 @@ public function getHeaders() {
 
 		$this->headers['Content-Security-Policy'] = $this->getContentSecurityPolicy()->buildPolicy();
 		$this->headers['Feature-Policy'] = $this->getFeaturePolicy()->buildPolicy();
+		$this->headers['X-Robots-Tag'] = 'noindex';
```

Comment:
```suggestion
		$this->headers['X-Robots-Tag'] = 'noindex, nofollow';
```

Will also avoid following links uselessly, cf https://developers.google.com/search/reference/robots_meta_tag The original robots.txt would have prevented to enter the site generally.